docs/security.md

```diff
@@ -157,5 +191,5 @@ dcos spark run --submit-args="\
 
  [11]: https://docs.mesosphere.com/1.9/overview/architecture/components/
  [12]: http://docs.oracle.com/javase/8/docs/technotes/tools/unix/keytool.html
- [13]: https://docs.mesosphere.com/1.10/security/#spaces
+ [13]: https://docs.mesosphere.com/1.10/security/ent/#spaces
```

Comment:
(OK. Here is the `[13]`. Should these links not all refer to `latest` instead of specific DC/OS versions?

Reply:
Fixed